### ISSUE TITLE
Review progress for the reference implementation

### ISSUE THREAD

**Issue by pablogsal** [gvanrossum/patma#163]
- [ ] Grammar changes
- [ ] Parser
- [ ] ASDL changes
- [ ] Other AST changes
- [ ] Builtin type changes in C
- [ ] Builtin type changes in Python
- [ ] Standard library changes
- [ ] Compiler changes & opcodes
- [ ] Symbol table changes
- [ ] VM changes (eval loop...)

**Comment by lysnikolaou** [comment]
I can help out with the review of the grammar and parser changes if that helps you, @pablogsal.

**Comment by brandtbucher** [comment]
Quick update: I'm making good progress with cleanup and plan on opening a draft PR against master before tomorrow.

**Comment by brandtbucher** [comment]
Then whoever wants to can start leaving more detailed comments (so they're all in one place).

**Comment by pablogsal** [comment]
> I can help out with the review of the grammar and parser changes if that helps you, @pablogsal.

Absolutely! I will do a pass but it would be great if you can give it a scan too :)

**Comment by brandtbucher** [comment]
https://github.com/python/cpython/pull/22917
